pkg/models/plan_general_characteristics.go

```diff
@@ -0,0 +1,138 @@
+	GeographiesTargetedTypes                  pq.StringArray `json:"geographiesTargetedTypes" db:"geographies_targeted_types"`
+	GeographiesTargetedTypesOther             *string        `json:"geographiesTargetedTypesOther" db:"geographies_targeted_types_other"`
+	GeographiesTargetedAppliedTo              pq.StringArray `json:"geographiesTargetedAppliedTo" db:"geographies_targeted_applied_to"`
+	GeographiesTargetedAppliedToOther         *string        `json:"geographiesTargetedAppliedToOther" db:"geographies_targeted_applied_to_other"`
+	GeographiesTargetedNote                   *string        `json:"geographiesTargetedNote" db:"geographies_targeted_note"`
+	ParticipationOptions                      *bool          `json:"participationOptions" db:"participation_options"`
+	ParticipationOptionsNote                  *string        `json:"participationOptionsNote" db:"participation_options_note"`
+	AgreementTypes                            pq.StringArray `json:"agreementTypes" db:"agreement_types"`
+	AgreementTypesOther                       *string        `json:"agreementTypesOther" db:"agreement_types_other"`
+	MultiplePatricipationAgreementsNeeded     *bool          `json:"multiplePatricipationAgreementsNeeded" db:"multiple_patricipation_agreements_needed"`
+	MultiplePatricipationAgreementsNeededNote *string        `json:"multiplePatricipationAgreementsNeededNote" db:"multiple_patricipation_agreements_needed_note"`
+
+	// Page 5
+	RulemakingRequired            *bool          `json:"rulemakingRequired" db:"rulemaking_required"`
+	RulemakingRequiredDescription *string        `json:"rulemakingRequiredDescription" db:"rulemaking_required_description"`
+	RulemakingRequiredNote        *string        `json:"rulemakingRequiredNote" db:"rulemaking_required_note"`
+	AuthorityAllowances           pq.StringArray `json:"authorityAllowances" db:"authority_allowances"`
+	AuthorityAllowancesOther      *string        `json:"authorityAllowancesOther" db:"authority_allowances_other"`
+	AuthorityAllowancesNote       *string        `json:"authorityAllowancesNote" db:"authority_allowances_note"`
+	WaiversRequired               *bool          `json:"waiversRequired" db:"waivers_required"`
+	WaiversRequiredTypes          pq.StringArray `json:"waiversRequiredTypes" db:"waivers_required_types"`
+	WaiversRequiredNote           *string        `json:"waiversRequiredNote" db:"waivers_required_note"`
+
+	// Meta
+	CreatedBy   *string    `json:"createdBy" db:"created_by"`
+	CreatedDts  *time.Time `json:"createdDts" db:"created_dts"`
+	ModifiedBy  *string    `json:"modifiedBy" db:"modified_by"`
+	ModifiedDts *time.Time `json:"modifiedDts" db:"modified_dts"`
+	Status      TaskStatus `json:"status" db:"status"`
+}
+
+// CalcStatus returns a TaskStatus based on how many fields have been entered in the PlanGeneralCharacteristics struct
+func (gc *PlanGeneralCharacteristics) CalcStatus() {
+
+	// TODO implement this. Doing it like this for now :)
+	gc.Status = TaskReady
```

Comment:
Is this necessary for this PR?

Reply:
I will change this comment! After writing this, I actually created EASI-2048, which should capture most of this work a bit more broadly (since I'm not a big fan of the way we have to do this now - all of the other `CalcStatus()` methods have a similar TODO note)